classes/publication/PKPPublication.inc.php

```diff
@@ -202,14 +213,24 @@ public function getAuthorString($userGroups)
      *
      * @return string
      */
-    public function getShortAuthorString($defaultLocale = null)
+    public function getShortAuthorString($defaultLocale = null, $includeInBrowseOnly = false)
     {
         $authors = $this->getData('authors');
 
         if (!$authors->count()) {
             return '';
         }
 
+        if ($includeInBrowseOnly) {
+            $authors = $authors->filter(function ($author, $key) {
+                return $author->getData('includeInBrowse');
+            });
+
+            $authors = $authors->sortBy(function ($author, $key) {
+                return $author->getData('seq');
+            });
+        }
```

Comment:
`$includeInBrowseOnly` is not used in this method and I think you can remove it.

Reply:
This could be deprecated due to code changes.

Reply:
I'm still not seeing anywhere in the code that uses `getShortAuthorString()` with the `includeInBrowseOnly` flag. Is this code used?